engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -356,6 +356,10 @@ def _feature_is_enabled_via_enable_key(
 
     def _feature_toggle_is_enabled(self, instance_info: GCOMInstanceInfo, feature_name: str) -> bool:
         """
+        NOTE: there are some edge-cases to consider if you decide to use this method
+        1. If a feature flag is enabled outside of the Admin UI (ex. via deployment_tools), this endpoint
+        will not return the ACTUAL runtime value that the Grafana stack itself is using/respecting
+
```

Comment:
I'm honestly considering getting rid of this client method (+ all the logic to handle the various feature toggle edge cases 😆)..

We got bit here because the `accessControlOnCall` feature toggle was "enabled" for all stacks in production, but this was done via `grafana/deployment_tools`. **However**, GCOM apparently doesn't return the value of this feature toggle if it is set in this manner (so we were basically considering RBAC as disabled for almost all stacks unless they had this feature flag value explicitly set in the Grafana Admin UI)

Reply:
I'm not sure if there are other implications but in principle I agree with getting rid of this.  GCOM only knows about overrides, the "default" state is in hosted-grafana config so that's why we weren't getting it returned here.

Reply:
Digging around a bit `accessControlOnCall` is available by asking the grafana instance instead on `/api/frontend/settings`